### DISCUSSION TITLE
Idea : Request to be V measured in TIOBE Index

### DISCUSSION THREAD

**Discussion by VitSimon** [vlang/v#20599] [Ideas]
Hi, I got an idea to ask TIOBE Company to start measure V lang in its [Tiobe Index](https://www.tiobe.com/tiobe-index/). I think it can help with making V more visible. Some companies are using this index for decision about theirs new products.

**Comment by Wajinn** [comment]
TIOBE index has a lot of background "politics" that people are not fully aware of, which affects who does or doesn't get on the list. The best people can do is email their boss (who decides) and present their case or ask for fair treatment.

**Comment by mkhan-akhan** [comment]
Have a look at TIOBE Index for April 2025. V has been positioned at 43, which I believe is quite amazing but not surprising for me.  V is a simple, neat and beautiful language.  TIOBE Index's section headed This Month's Changes in the Index states: "Clemence suggested to add the V programming language to the TIOBE index because it meets all criteria to be listed. The V programming language enters the TIOBE index at position #43 and is thus considered a promising language. "

**Comment by medvednikov** [comment]
That's pretty cool!

![telegram-cloud-photo-size-4-6030782460893185347-y](https://github.com/user-attachments/assets/1710bb0c-c929-4f28-906b-49be7c77bbc8)
